client/my-sites/themes/index.node.js

```diff
@@ -15,9 +15,13 @@ export default function( router ) {
 
 	if ( config.isEnabled( 'manage/themes' ) ) {
 		if ( config.isEnabled( 'manage/themes-ssr' ) ) {
-			router( `/design/:vertical(${ verticals })?/:tier(free|premium)?$`, fetchThemeDataWithCaching, loggedOut, makeLayout );
-			router( `/design/:vertical(${ verticals })?/:tier(free|premium)?`, fetchThemeData, loggedOut, makeLayout );
-			router( `/design/:vertical(${ verticals })?/:tier(free|premium)?/filter/:filter`, fetchThemeData, loggedOut, makeLayout );
+			router( `/design/:vertical(${ verticals })?/:tier(free|premium)?`, fetchThemeDataWithCaching, loggedOut, makeLayout );
+			router(
+				`/design/:vertical(${ verticals })?/:tier(free|premium)?/filter/:filter`,
+				fetchThemeDataWithCaching,
+				loggedOut,
+				makeLayout
+			);
 			router( '/design/upload/*', makeLayout );
 			router( '/design/*', fetchThemeData, loggedOut, makeLayout ); // Needed so direct hits don't result in a 404.
```

Comment:
Hmm, good point. Also, looks like this route never kicks in, because [this one](https://github.com/Automattic/wp-calypso/pull/12592/files/183475662715f7463499c74f506c3dbfb21c9ff6#diff-750f44a78dda5cf536254ebb29b69f5eR18) does first. I'll see what happens if I remove it.